DGWebApi/Controllers/SatelliteController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using DGWebApi.Models;
+using Microsoft.AspNetCore.Mvc;
+using DGWebApi.Services.Interfaces;
+
+namespace DGWebApi.Controllers;
```

Comment:
Choose either block-scoped or file-scoped namespace definitions - each controller is using a different style

Reply:
done